packages/app/components/drop/drop-form.tsx

```diff
@@ -31,6 +30,7 @@ import { useRouter } from "app/navigation/use-router";
 import { getTwitterIntent, getUserDisplayNameFromProfile } from "app/utilities";
 
 import { useFilePicker } from "design-system/file-picker";
+import { ScrollView } from "design-system/scroll-view";
```

Comment:
We can switch back to `@showtime-xyz/universal.scroll-view` once we release the new version of `universal.scroll-view` via Bit

Reply:
Done